cyber/tools/cyber_monitor/general_channel_message.h

```diff
@@ -72,6 +72,8 @@ class GeneralChannelMessage : public GeneralMessageBase {
 
   double frame_ratio(void) override;
 
+  double band_width(void) override;
```

Comment:
now band_width function is not necessary for redefinition because its implementation is the same with parent class.